CMake/SofaPython3Tools.cmake

```diff
@@ -85,13 +97,9 @@ function(SP3_add_python_module)
         set(A_TARGET ${MODULE_NAME})
     endif()
 
-    if (A_PYTHON_VERSION)
-        set(PYBIND11_PYTHON_VERSION ${A_PYTHON_VERSION})
-    endif()
-
     set(PYBIND11_CPP_STANDARD -std=c++17)
 
-    find_package(pybind11 CONFIG REQUIRED)
+    find_package(pybind11 CONFIG QUIET REQUIRED)
```

Comment:
Why the QUIET here? I don't see any condition  ran on PYBIND11_FOUND later on

Reply:
QUIET is there to shut up the output of `pybind11Config.cmake`, the REQUIRED is still there.